.circleci/config.yml

```diff
@@ -0,0 +1,101 @@
+aliases:
+  - &restore-cache
+    keys:
+      - dependencies-{{ .Branch }}-{{ checksum "package.json" }}
```

Comment:
Can you prefix all the keys with `v1-` so we can easily manually bust the cache if we need to?

I'm also concerned about sharing node_modules caches across npm versions. In particular, Node 4 comes with npm 2 which doesn't have the deduping functionality of npm 3+.

What if we didn't cache node_modules for react-native but continued to cache them for the website and danger (where we use just a single Node + npm version)?